main.go

```diff
@@ -222,6 +223,29 @@ func main() {
 		}
 	}
 
+	// Remove untagged images, if any
+	var outbuf bytes.Buffer
+	cmd = exec.Command("docker", "images", "-q", "-f", "dangling=true")
+	cmd.Stdout = &outbuf
+	cmd.Stderr = os.Stderr
+	trace(cmd)
+	err = cmd.Run()
+	if err != nil {
+		os.Exit(1)
+	}
+
+	if outbuf.Len() > 0 {
+		images := strings.Split(strings.TrimSpace(outbuf.String()), "\n")
+		cmd = exec.Command("docker", append([]string{"rmi"}, images...)...)
```

Comment:
Tried that. It doesn't seem to be possible to add a single parameters as well as an array slice for a varargs parameter (`too many arguments in call to exec.Command`). 

Not a Go expert though.